src/main/resources/connector.xml

```diff
@@ -17,21 +17,13 @@
  ~  under the License.
 -->
 <connector>
-    <component name="twitter" package="org.wso2.carbon.connector">
+    <component name="twitter" package="org.wso2.carbon.connectors">
```

Comment:
Do we need to change this?

Reply:
Without that 
`ERROR {SynapseConfiguration} - Unable to deploy synapse import:{org.wso2.carbon.connectors}twitter. Required library not found.`
issue arises

Reply:
We usually have `org.wso2.carbon.connector` as the package in all the connectors. It seems like an issue with the import configs in `EI_HOME/repository/deployment/server/synapse-configs/default/imports/`.